vdr-osdteletext: package teletext2 font and fix blank lines

tmn505 opened the pull request:
Package the teletext2.ttf font, on which osdteletext relies now for displaying characters. This follows the request in issue #175. The update also incorporates fix for characters alignment which is seen as blank horizontal lines.

M-Reimer commented:
@CReimer I'm unsure if it's a "clean" way to package the TTF file with osdteletext. Would a separate font package be the better way?

tmn505 commented:
That was my first approach but realised that unlikely there will be any packages depending on that font. If You'll request, I'll make it a split package.

tmn505 commented:
@M-Reimer 
As @CReimer didn't comment on the issue I adjusted the commit to Your preferences. Please review.

tmn505 commented:
Updated PR. Pushed separate packages instead of a split one.

tmn505 commented:
I tested it today and seems like osdteletext can use already installed ttf-font (which is pulled by vdr). It also has settings for setting which font to use. During the test I didn't spot problems with displaying letters when using DejaVu font, there were small issues with tables but nothing that made them unreadable. It is more of an cosmetic issue when not using teletext2 font, hence making it optional.
Updated the commits with expanding a bit the commit title and messages, and fixed .SRCINFO for ttf-teletext2.
